source/common/CMakeLists.txt

```diff
@@ -24,8 +24,8 @@ if (NOT DEFINED UMF_REPO)
 endif()
 
 if (NOT DEFINED UMF_TAG)
-    # v0.9.x 23.08.2024: Merge pull request #696 ...
-    set(UMF_TAG 3c340e61c197f4f9e29abd947f90ce27c571433e)
+    # 02 Sept 2024: Merge pull request #701
+    set(UMF_TAG 72e22a95f7b8a932e4eebb49e31435abaefd3b06)
```

Comment:
bumping UMF version in this PR is done on purpose..?

Reply:
yes, I want UR to compile on SLES
It needs some warning fixes and UMF with fix

Reply:
FYI, your fix will be included as part of this PR: https://github.com/oneapi-src/unified-runtime/pull/2012